registry.json

```diff
@@ -2880,5 +2880,37 @@
       "source": "https://github.com/hashgraph",
       "documentation": "https://docs.hedera.com"
     }
+  },
+  {
+    "id": "agoric",
+    "name": "Agoric",
+    "displayName": "Agoric",
+    "coinId": 564,
+    "symbol": "BLD",
```

Comment:
According to https://github.com/satoshilabs/slips/blob/master/slip-0044.md , symbol is 'AG', please clarify.

Reply:
The information was taken from https://github.com/cosmos/chain-registry/blob/12a99dc31656b767fabe474f01468f6d2bc62c36/agoric/assetlist.json#L6

There is no coin with symbol `AG` in Agoric.